components/plan-build/bin/hab-plan-build.sh

```diff
@@ -1977,7 +1977,18 @@ do_default_build_config() {
   _do_copy_templates "config"
   _do_copy_templates "config_install"
   if [[ -d "$PLAN_CONTEXT/hooks" ]]; then
-    cp -r "$PLAN_CONTEXT/hooks" "$pkg_prefix"
+    mkdir -p "$pkg_prefix"/hooks
```

Comment:
We should declare `local target file` in this function to keep those variables in the scope of this function.